presto-router/src/main/java/com/facebook/presto/router/scheduler/SchedulerType.java

```diff
@@ -16,5 +16,7 @@
 public enum SchedulerType
 {
     RANDOM_CHOICE,
+    WEIGHTED_RANDOM_CHOICE,
```

Comment:
nit: in alphabetical order?

Reply:
Good point! That'll make it clearer.
I updated the order here.